examples/more/external-project-add/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+cmake --build ${HOME}/temp/external-project-add/build
```

Comment:
I'm not sure this `${HOME}/temp/` folder is the right way to go, we ideally want everything to relative to this repo, it looks like this could end up elsewhere?